RoboFile.php

```diff
@@ -282,6 +282,21 @@ function testFailedIntegration() {
     return $this->_exec('vendor/bin/codecept run integration -g failed');
   }
 
+  function containerDump() {
+    $this->say('Deleting DI Container');
+    $this->_exec('rm ./lib/DI/CachedContainer.php');
```

Comment:
This step can end up with an error if the file is not present. Could we suppress the error? It looks like the whole command failed.